### ISSUE TITLE
Listing of DNS records seems to be unstable

### ISSUE THREAD

**Issue by MyCADDev** [drifting-in-space/plane#265]
I just tried this with latest pull and install of cli 

Created two backends

```
% plane-cli spawn plane.test ghcr.io/drifting-in-space/demo-image-drop-four
Backend scheduled.
URL: https://64becea6-7e02-472e-bc1e-8319363843d5.plane.test
Drone: 6c12439f-6e48-4a4a-96b1-489be751ed56
Backend ID: 64becea6-7e02-472e-bc1e-8319363843d5

% plane-cli spawn plane.test ghcr.io/drifting-in-space/demo-image-drop-four
Backend scheduled.
URL: https://03b6ca25-b8f1-4519-952d-56d845474434.plane.test
Drone: 6c12439f-6e48-4a4a-96b1-489be751ed56
Backend ID: 03b6ca25-b8f1-4519-952d-56d845474434
```

List DNS is always showing one or the other - never both
```
% plane-cli list-dns
Found 1 DNS records:
64becea6-7e02-472e-bc1e-8319363843d5.plane.test	A	172.16.238.3

% plane-cli list-dns
Found 1 DNS records:
03b6ca25-b8f1-4519-952d-56d845474434.plane.test	A	172.16.238.3

% plane-cli list-dns
Found 1 DNS records:
03b6ca25-b8f1-4519-952d-56d845474434.plane.test	A	172.16.238.3

% plane-cli list-dns
Found 1 DNS records:
03b6ca25-b8f1-4519-952d-56d845474434.plane.test	A	172.16.238.3

% plane-cli list-dns
Found 1 DNS records:
64becea6-7e02-472e-bc1e-8319363843d5.plane.test	A	172.16.238.3
```

**Comment by MyCADDev** [comment]
This may be a non-issue / low priority issue as both URLs work in Firefox so the DNS is working correctly - its only the CLI which is listing only one or the other

**Comment by paulgb** [comment]
Ok, I figured out what the problem is. I have a fix that works (#268), but it's not ready to be merged yet because I need to think through the path to do a zero-downtime deploy.

**Comment by paulgb** [comment]
Closed by #341
